### PR TITLE
DTGB-580: Added check to fix error.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.
- [ ] I have ran gulp axe on the compiled files and found no errors.

### CODE REVIEW

**Review by MPParsley — CHANGES_REQUESTED** [review]
This will result in a notice;

Notice: Undefined index: #object in gent_base_preprocess_node__outdoor_location__teaser()

**Comment by meirege** [comment]
@MPParsley Hoe check ik dan best hierop zodat ik geen notice krijg?

**Comment by MPParsley** [comment]
Je kan hiervoor isset() gebruiken.

File: gent_base.theme [hunk 1]
```diff
@@ -502,7 +502,14 @@ function gent_base_preprocess_node(&$variables) {
...
 function gent_base_preprocess_node__outdoor_location__teaser(&$variables) {
   // Get the field value.
-  $related_terrains = $variables['elements']['field_related_terrains']['#object'];
+  if (isset($variables['elements']['field_related_terrains']['#object'])) {
+    $related_terrains = $variables['elements']['field_related_terrains']['#object'];
+  }
+
+  if (!$related_terrains) {
+    return;
+  }
```
Comment (MPParsley) [inline gent_base.theme:507]:
```
  if (!isset($variables['elements']['field_related_terrains']['#object'])) {
    return;
  }
  $related_terrains = $variables['elements']['field_related_terrains']['#object'];

```

Comment (meirege) [reply gent_base.theme:507]: Fixed.

**Review by MPParsley — APPROVED** [review]

File: gent_base.theme [hunk 2]
```diff
@@ -502,7 +502,15 @@ function gent_base_preprocess_node(&$variables) {
  */
 function gent_base_preprocess_node__outdoor_location__teaser(&$variables) {
   // Get the field value.
+  if (!isset($variables['elements']['field_related_terrains']['#object'])) {
+    return;
+  }
+
   $related_terrains = $variables['elements']['field_related_terrains']['#object'];
+  if (!$related_terrains) {
```
Comment (MPParsley) [inline gent_base.theme]: deze lijkt me niet meer nodig?

**Pull request merged**